office-products/Export success/src/app/app.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import {

```

Comment:
**Folder name** 
**sample** - Common/ Load PDF, Excel, PPT file types
**web service** - Common/ web service 

------------
office-products/Export success/src/assets/.gitkeep - what is the use of this file?